core/risk/engine_test.go

```diff
@@ -482,24 +482,24 @@ func TestMarginWithNoOrdersOnBook(t *testing.T) {
 			positionSize:   9,
 			buyOrders: []*risk.OrderInfo{
 				{
-					Size:          3,
+					Remaining:     3,
```

Comment:
VERY much pedantry and nit-picky, but I've encountered it a couple times recently and it got on my nerves a bit: is this `Remaining` the visible remaining size on the book, or `TrueRemaining` (ie including the invisible part of iceberg orders)?

Reply:
Well it's whatever the caller of the endpoint chooses is to be, but I guess `TrueRemaining` might be better to encourage setting it as we'd anticipate.

Reply:
Renamed now